modules/era/lua_dynamis/mobs/era_xarcabard_mobs.lua

```diff
@@ -623,5 +623,7 @@ xi.dynamis.onFightSatellite = function(mob, target)
 
     if not GetMobByID(mob:getLocalVar("ParentID")) then -- Despawn if Animated Parent Dies
         DespawnMob(mob:getID())
+    elseif GetMobByID(mob:getLocalVar("ParentID")) and GetMobByID(mob:getLocalVar("ParentID")):getHP() == 0 then
```

Comment:
This should be able to be short-circuited into one line?
```lua
if not GetMobByID(mob:getLocalVar("ParentID")) or GetMobByID(mob:getLocalVar("ParentID")):getHP() == 0 then

Reply:
Changed